### ISSUE TITLE
Spamtrap

### ISSUE THREAD

**Issue by rafaelg80** [AfterShip/email-verifier#123]
Hello, does this catch spamtrap emails like Reoon and other services do?

Thanks,

**Comment by NeoCN** [comment]
No, and it's unlikely to. Spamtrap detection isn't something you can determine from protocol-level signals — services like Reoon rely on proprietary, continuously-updated databases of known trap addresses and domains, plus engagement data from mail streams.

  This library does protocol- and metadata-level checks only: syntax, MX, SMTP `RCPT TO`, disposable domains, free providers, role accounts. Everything ships as static metadata in the repo, which is fundamentally the wrong shape for trap data — it goes stale fast and publishing a trap list would defeat its purpose.

  If you need it, pair this library with a commercial reputation service.

  Closing as out of scope.
